apps/docs/src/components/code/PopupLayout.tsx

```diff
@@ -81,11 +81,7 @@ export const PopupLayout = ({ scope, code }: PopupLayoutProps) => {
       />
 
       {/* Preview Section */}
-      <div
-        className={clsx(
-          "p-md pt-lg bg-bgPage border rounded-inherit flex items-center justify-center",
-        )}
-      >
+      <div className={"p-md flex items-center justify-center"}>
```

Comment:
`clsx` is no longer used (can't comment it on the line)
```suggestion
      <div className="p-md flex items-center justify-center">
```